aws-proofs/vm-steps.sh

```diff
@@ -82,14 +82,15 @@ export SKIP_DUPLICATED_PROOFS=${INPUT_SKIP_DUPS}
 
 FAIL=0
 
-cd l4v
+l4v_dir="$PWD/l4v"
```

Comment:
The convention in the repo is to use upper case var names in the shell scripts.

```suggestion
L4V_DIR="$PWD/l4v"
```